packages/fast-usdc/test/fixtures.ts

```diff
@@ -166,3 +161,9 @@ export const intermediateRecipient: ChainAddress = harden({
   value: 'noble1test',
   encoding: 'bech32',
 });
+
+export const settlementAddress: ChainAddress = harden({
+  chainId: 'agoric-3',
+  encoding: 'bech32' as const,
+  value: 'agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek',
```

Comment:
random? if it's not encoding any data let's put something descriptive in it

Reply:
These was extracted from existing code on `master`. I don't think it's encoding anything - it's supposed to represent an LCA. Maybe better to use `makeTestAddress()`.

Reply:
Since this is in a `fixtures` module I'd propose a fixed value. `makeTestAddress` values could collide. We might also consider having a global counter so each call produces a unique address. That will help prevent using magic strings: when we need values to match put them in a constant.